index.js

```diff
@@ -259,9 +260,12 @@ logger.setLevel = function setLoggingLevel(level) {
     return;
   }
 
-  var intLevel = parseInt(level);
-  if (!isNaN(intLevel)) loggingLevel = level;
-  else throw new Error('Invalid logging level');
+  var intLevel = parseInt(level, 10);

```

Comment:
`parseInt(s, 10)` always returns a number, so there is no need to type check here. It was causing tests to fail.